backend/test/v2/api/integration_suite_test.go

```diff
@@ -88,10 +87,15 @@ var _ = BeforeSuite(func() {
 		newRecurringRunClient = func() (*apiserver.RecurringRunClient, error) {
 			return apiserver.NewKubeflowInClusterRecurringRunClient(*config.Namespace, *config.DebugMode)
 		}
-	} else if *config.MultiUserMode {
-		logger.Log("Creating API Clients for Multi User Mode")
-		defaultUserToken = test_utils.CreateUserToken(k8Client, *config.Namespace, *config.DefaultServiceAccountName)
-		userToken = test_utils.CreateUserToken(k8Client, *config.UserNamespace, *config.UserServiceAccountName)
+	} else if *config.MultiUserMode || config.AuthToken != nil {
+		if *config.MultiUserMode {
+			logger.Log("Creating API Clients for Multi User Mode")
+			userToken = test_utils.CreateUserToken(k8Client, *config.UserNamespace, *config.UserServiceAccountName)
+		} else if *config.AuthToken != "" {
+			logger.Log("Creating API Clients With Auth Token")
+			userToken = *config.AuthToken
+		}
+		logger.Log("User Token: %s", userToken)
```

Comment:
It's benign since these are dev environments but it doesn't seem "right" to log a token lol